src/branch.rs

```diff
@@ -151,6 +161,7 @@ impl<'repo> Drop for Branches<'repo> {
 
 #[cfg(test)]
 mod tests {
+    use crate::Branch;
     use crate::BranchType;
```

Comment:
Tipp (since you implied you'd be somewhat new to rust): you can combine imports into a single line. In this case, for example, you could write `use crate::{Branch, BranchType}`. But I don't know what's the maintainers' preference.

Reply:
yes, I'm still unsure about these style decisions. thanks for the feedback anyway :)